OutOfSchool/OutOfSchool.WebApi/Services/LicenseApprovalNotification/LicenseApprovalNotificationService.cs

```diff
@@ -39,7 +39,7 @@ await notificationService
 
     public async Task<IEnumerable<string>> GetNotificationsRecipientIds(NotificationAction action, Dictionary<string, string> additionalData, Guid objectId)
     {
-        // TODO Add filter for OtgAdmin when he will be created, and delete filter for TechAdmin after that
+        // TODO Add filter for AreaAdmin when he will be created, and delete filter for TechAdmin after that
```

Comment:
I think you should resolve this TODO because we already have the area admin. Please, contact to @DmyMi, if techadmin doesn't need to get the notifications already.